BiometricLockerTests/BiometricLockerTests.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  BiometricLockerTests.swift
+//  BiometricLockerTests
+//
+//  Created by Igor Ranieri on 16.04.18.
+//  Copyright © 2018 Bakken&Bæck. All rights reserved.
+//
+
+import XCTest
+import LocalAuthentication
+@testable import BiometricLocker
+
+class BiometricLockerTests: XCTestCase {
+
+    override func setUp() {
+        super.setUp()
+
+        // Since lockers are not namespaced or anything, if you have multiple lockers, they'll all use the same UserDefaults date value.
+        // To ensure we're always starting from scratch, create one and unlock to clear the UserDefaults value. Will improve on this.
```

Comment:
Could make that `UserDefaults` value mockable, but doesn't need to happen now.

Reply:
Not a fan of mocks. 😅

Reply:
Fair enough, though we should probably have a longer discussion offline about that in terms of partial mocks. Full mocks definitely can have sharp edges, but I've found partial mocks to be very helpful.